src/dev/flang/be/interpreter/Intrinsics.java

```diff
@@ -75,6 +75,22 @@ public class Intrinsics extends ANY
   }
 
 
+  /*------------------------------  enums  ------------------------------*/
+
+
+  enum IntrinsicError
```

Comment:
Please add a comment here. 

`IntrinsicError` is maybe not the best name, there is nothing wrong with the intrinsic mechanism here. I think these are declared in _errno.h_, so `ErrNo` or `SystemErrNo` would maybe be better names.

In a second patch, I think we will need a counterpart to this in _fz/intrinsic_error.fz_ or similar to map these back to human-readable error messages.